app/assets/javascripts/discourse/app/components/user-menu/default-reviewable-item.js

```diff
@@ -0,0 +1,27 @@
+import GlimmerComponent from "discourse/components/glimmer";
+import I18n from "I18n";
+
+export default class UserMenuReviewableItem extends GlimmerComponent {
+  get actor() {
+    const flagger = this.reviewable.flagger_username;
+    if (flagger) {
+      return flagger;
+    } else {
+      return I18n.t("user_menu.reviewable.deleted_user");
+    }
+  }
+
+  get description() {
+    return I18n.t("user_menu.reviewable.default_item", {
+      reviewable_id: this.reviewable.id,
+    });
+  }
+
+  get icon() {
+    return "flag";
+  }
+
+  get reviewable() {
+    return this.args.item;
+  }
+}
```

Comment:
```suggestion
export default class UserMenuReviewableItem extends GlimmerComponent {
  constructor(owner, args) {
    super(...arguments);
    this.reviewable = args.item;
  }

  get actor() {
    const flagger = this.reviewable.flagger_username;
    if (flagger) {
      return flagger;
    } else {
      return I18n.t("user_menu.reviewable.deleted_user");
    }
  }

  get description() {
    return I18n.t("user_menu.reviewable.default_item", {
      reviewable_id: this.reviewable.id,
    });
  }

  get icon() {
    return "flag";
  }
}
```

Minor but you don't need to define a getter just to assign a property from `args`.